CHANGELOG.md

```diff
@@ -16,6 +16,12 @@ should change the heading of the (upcoming) version to include a major version b
 
 -->
 
+# 5.18.5
+
+## @rjsf/antd
+
+- Fix disabled property of options in CheckboxesWidget and RadioWidgep ([#4216](https://github.com/rjsf-team/react-jsonschema-form/pull/4216))
+
```

Comment:
@KingMario if you resolve the conflict in this file then I will approve and merge

Reply:
Sorry for being late. The conflit is resolved.